backend/src/WebApi/Controllers/CommissionController.cs

```diff
@@ -23,10 +25,25 @@ public CommissionController(ILogger<CommissionController> logger, ICommissionSer
         [ProducesResponseType(StatusCodes.Status400BadRequest)]
         [ProducesResponseType(StatusCodes.Status403Forbidden)]
         [Authorize(Policy = "AdminOnly")]
-        public async Task<SetCommissionDTO> GetCommissionAsync()
+        public async Task<ActionResult<SetCommissionDTO>> GetCommissionAsync()
         {
-            decimal commissionValue = await _commissionService.GetCommissionAsync();
-            return new SetCommissionDTO(commissionValue);
+            if (User.IsAdmin())
```

Comment:
Do czego jest to sprawdzenie czy User jest adminem? Do tego endpointu może wejść tylko admin, przez to że jest [Authorize(Policy = "AdminOnly")].